bundle/android/strings.xml

```diff
@@ -63,7 +63,6 @@
     <string name="gutenberg_native_build_layouts" tools:ignore="UnusedResources">Build layouts</string>
     <string name="gutenberg_native_button_link_url" tools:ignore="UnusedResources">Button Link URL</string>
     <string name="gutenberg_native_button_position" tools:ignore="UnusedResources">Button position</string>
-    <string name="gutenberg_native_cancel" tools:ignore="UnusedResources">Cancel</string>
```

Comment:
- Referenced in [`block-experiments/blocks/layout-grid/src/grid/variation-control/index.native.js`](https://github.com/Automattic/block-experiments/blob/00dbb963f1615d9843f0d8a8479b585c7a0108e1/blocks/layout-grid/src/grid/variation-control/index.native.js#L160-L162)
- [Translated in Gutenberg GlotPress project](https://translate.wordpress.org/projects/wp-plugins/gutenberg/stable/en-gb/default/?filters%5Bterm%5D=Cancel&filters%5Bterm_scope%5D=scope_any&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)